content/en/docs/contributing/localization.md

```diff
@@ -93,6 +94,31 @@ path would become `/ja/docs/some-page` when rendered from a Japanese page.
 
 {{% /alert %}}
 
+### Link definition labels {#link-labels}
+
+Do **not** translate [labels] of markdown [link definitions][]. Instead, rewrite
+the label as translated link text. For example, consider the following markdown:
+
+```markdown
+[Hello], world! Welcome to the [OTel website][].
+
+[hello]: https://code.org/helloworld
```

Comment:
Does this `hello` need to be capitalized to match the in-line reference? I'm not sure if capitalization matters...

Reply:
No, label-matching is case insensitive; see https://spec.commonmark.org/0.31.2/#example-539

> Matching is case-insensitive:
>
> [Example 539](https://spec.commonmark.org/0.31.2/#example-539)